src/common/utxobased/keymanager/coins/bitcoin.ts

```diff
@@ -6,30 +6,30 @@ export class Bitcoin implements Coin {
   coinType = 0
 
   mainnetConstants = {
-    messagePrefix: '\x18Bitcoin Signed Message:\n',
-    wif: 0x80,
-    legacyXPriv: 0x0488ade4,
-    legacyXPub: 0x0488b21e,
-    wrappedSegwitXPriv: 0x049d7878,
-    wrappedSegwitXPub: 0x049d7cb2,
-    segwitXPriv: 0x04b2430c,
-    segwitXPub: 0x04b24746,
-    pubkeyHash: 0x00,
-    scriptHash: 0x05,
-    bech32: 'bc'
+    messagePrefix: ['\x18Bitcoin Signed Message:\n'],
+    wif: [0x80],
+    legacyXPriv: [0x0488ade4],
+    legacyXPub: [0x0488b21e],
+    wrappedSegwitXPriv: [0x049d7878],
+    wrappedSegwitXPub: [0x049d7cb2],
+    segwitXPriv: [0x04b2430c],
+    segwitXPub: [0x04b24746],
+    pubkeyHash: [0x00],
+    scriptHash: [0x05],
+    bech32: ['bc']
```

Comment:
These should include the legacy constants as the second element of each array. It's apart of the requirement for the task: 

> The datatype for each prefix field must be an array where the first element in the array is the default prefix for creating addresses.